.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+repos:
+-   repo: https://github.com/pre-commit/mirrors-yapf    # To format the code to conform YAPF
+    rev: v0.30.0
+    hooks:
+    - id: yapf
+      args: ['--style=.style.yapf', '--recursive', '--in-place']
```

Comment:
This is a different style than we've been using, I would suggest remaining on the google style.

Reply:
Nevermind, this is referncing a style file which is pulling google style. all good